app/onepassword_events_api/appserver/static/javascript/components/wizard.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+
+import { onepassword_name_space } from "../views/setup_page";
+
+const VERSION = "1.6.0";
```

Comment:
I think we should consider moving these constants out of the wizard and into setup or some other config place so we can import them throughout the app (rather than relying on updating the build steps to increment in so many places)